android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactory.java

```diff
@@ -164,7 +165,7 @@ private JSONObject getRepeatingGroupCountObj(WidgetArgs widgetArgs) throws JSONE
 
     @Override
     public Set<String> getCustomTranslatableWidgetFields() {
-        return new HashSet<>();
+        return new HashSet<>(Collections.singleton("reference_edit_text_hint"));
```

Comment:
`new HashSet<>(Arrays.asList("reference_edit_text_hint"));` use this instead to allow easier addition of other fields in the future.

Reply:
fixed.